docs/platform/howto/saml/add-identity-providers.md

```diff
@@ -9,19 +10,12 @@ provider (IdP).
 
 To set up single sign-on through an IdP for your organization:
 
-1. Add the identity provider in the [Aiven
-    Console](https://console.aiven.io/) .
-1. Configure SAML on your IdP.
-1. Finalize the setup in the Aiven Console using information from your
-    IdP.
-1. Link your users to the identity provider.
-
 ## Step 1. Add the IdP in the Aiven Console {#add-idp-aiven-console}
 
```

Comment:
Should we include a link to the Aiven Console here?

Reply:
I have no made a habit of adding this. Maybe something to discuss with the team.